pkg/kubelet/server/server.go

```diff
@@ -317,18 +326,28 @@ func (s *Server) InstallAuthFilter() {
 
 		// Get authorization attributes
 		attrs := s.auth.GetRequestAttributes(info.User, req.Request)
+		var allowed bool
+		var msg string
+		for _, attr := range attrs {
+			decision, _, err := s.auth.Authorize(req.Request.Context(), attr)
+			if err != nil {
```

Comment:
Is this basically only expected for 50X errors? I'm trying to decide if it's worth trying the other attributes in the error case...

One reason to try the fallback is in case the decision is cached.

Reply:
yeah the reason that I kept the fallback in was that the second decision might be cached. I am torn on this as well, one would expect the second call to fail too, since the only case I think we fail (error) is due to internal errors.

Reply:
Yeah, I think what you have here is fine.